waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/service/issuers/IssuerDataTransferObject.kt

```diff
@@ -24,13 +26,20 @@ data class IssuerDataTransferObject(
     )
 
     companion object {
-        fun default(wallet: UUID) = IssuerDataTransferObject(
-            wallet = wallet,
-            did = "did:web:walt.id",
-            description = "walt.id issuer portal",
-            uiEndpoint = "https://portal.walt.id/credentials?ids=",
-            configurationEndpoint = "https://issuer.portal.walt.id/.well-known/openid-credential-issuer",
-            authorized = false,
-        )
+        fun default(wallet: UUID): IssuerDataTransferObject? {
+            val defaultGenerationConfig by lazy { ConfigManager.getConfig<RegistrationDefaultsConfig>() }
+            return if (defaultGenerationConfig.defaultIssuerConfig != null) {
+                IssuerDataTransferObject(
+                    wallet = wallet,
+                    did = defaultGenerationConfig.defaultIssuerConfig!!.did,
```

Comment:
avoid using the (!!) operator , it can lead to `NullPointerException` if the value is null. Instead, we can use safe calls `(?.)` or providing default values

Reply:
Code updated, please check now.